lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -3,59 +3,72 @@
 ---
 _exclude: [N3k, N5k, N6k, N8k, N9k, ios_xr]
 
+# For the below name, shutdown commands we are going to use show running-config
+# bridge-domain cli which displays as below
+#
+# system bridge-domain 100-110
+#
+# bridge-domain 100-110
+#
+# bridge-domain 100
+#   shutdown
+#   fabric-control
+#   name PepsiCo
+# bridge-domain 101
+# bridge-domain 102
+# ...
+# bridge-domain 108
+#   shutdown
+#   name Amazon
+# bridge-domain 109
+#   shutdown
+# bridge-domain 110
+#   name Microsoft
+
+_template:
+  get_command: "show running-config bridge-domain"
+  get_context: ['/^bridge-domain <bd>\s$/']
+  set_context: ["bridge-domain <bd>"]
+
 all_bds:
   multiple: true
-  get_command: "show running-config bridge-domain"
+  get_context: ~
+  set_context: ~
```

Comment:
Yes after adding the optional for set_context this is not required.
Removed it.